### PR TITLE
Oms Extension test scripts

### PR DESCRIPTION
**Pull request by jamvar**
Updated similar to docker tests.

### CODE REVIEW

**Comment by jamvar** [comment]
All code refactor are according to @hestolz changes in docker test scripts and considering the addition of LAD side by side of Omsagent tests in pipeline

File: OmsAgent/extension-test/oms_extension_tests.py [hunk 1]
```diff
@@ -280,6 +304,80 @@ def create_vm_and_install_extension():
                             <td><span style='background-color: red; color: white'>Install Failed</span></td>"""
     return message
 
+def create_vm_and_install_old_extension():
+    """Create vm and install a specific version of the extension, returning HTML results."""
+
+    message = ""
+    for distname, image in images.iteritems():
+        update_option = '--version {0} --no-auto-upgrade'
```
Comment (hestolz) [inline OmsAgent/extension-test/oms_extension_tests.py]: doesn't seem like you ever .format() to insert the version

File: OmsAgent/extension-test/oms_extension_tests.py [hunk 2]
```diff
@@ -51,14 +51,27 @@
            '$ python -u oms_extension_tests.py length [image...]'))
```
Comment (hestolz) [inline OmsAgent/extension-test/oms_extension_tests.py]: could update help message to reflect new options

File: OmsAgent/extension-test/oms_extension_tests.py [hunk 3]
```diff
@@ -51,14 +51,27 @@
...
+is_instantupgrade = sys.argv[1] == 'instantupgrade'
+
 
 if len(sys.argv) > 2:
-    if sys.argv[2] == 'verbose':
+    if sys.argv[2] == 'debug':
         vms_list = sys.argv[3:]
-        runwith = '--verbose'
+        runwith = '--debug'
+    elif sys.argv[3] == 'debug':
```
Comment (hestolz) [inline OmsAgent/extension-test/oms_extension_tests.py]: @jamvar when would debug ever be sys.argv[3]? My understanding of sysargv:
0 = oms_extension_test.py
1 = one of [short, long, autoupgrade, instantupgrade]
2 = either debug, first specified image, or nothing
3 = either first specified image (if debug), second image (no debug) or nothing
4 = either second specified image (if debug), third image (no debug) or nothing
etc.

File: OmsAgent/extension-test/oms_extension_tests.py [hunk 4]
```diff
@@ -419,7 +517,7 @@ def reinstall_extension():
         append_file('omsfiles/omsresults.html', html_open)
         log_open.close()
         html_open.close()
-        status = open('omsfiles/omsresults.status')
+        status = open('omsfiles/omsresults.status').read()
```
[the remark this answers is not in the document]
Comment (jamvar) [reply OmsAgent/extension-test/oms_extension_tests.py]: yes

**Pull request merged**